api/CloseEvent.json

```diff
@@ -0,0 +1,147 @@
+{
+  "api": {
+    "CloseEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "8",
+            "notes": "Prior to Firefox 8, the WebSocket close event was sent to the listener as a simple event. Prior to Firefox 12, the close code CLOSE_NORMAL was used when the channel was closed due to an unexpected error or unspecified error conditoin."
```

Comment:
A few possible improvements to the notes here:

* Split "Prior to Firefox 8" and "Prior to Firefox 12" into separate notes (Firefox 12 note first)
* Code tags: `<code>CLOSE_NORMAL</code>`
* Typo: "conditoin" ➡️ "condition"

Reply:
Thanks, fixed in 20a7bf6 now.